geopandas/tools/_show_versions.py

```diff
@@ -22,6 +22,53 @@ def _get_sys_info():
     return dict(blob)
 
 
+def _get_C_info():
+    """Information on system PROJ, GDAL, GEOS
+    Returns
+    -------
+    c_info: dict
+        system PROJ information
+    """
+    try:
+        import pyproj
+        from pyproj.exceptions import DataDirError
+        proj = pyproj.proj_version_str
+        try:
+            proj_dir = pyproj.datadir.get_data_dir()
+        except DataDirError:
+            proj_dir = None
+    except ImportError:
+        proj = None
+        proj_dir = None
+
+    try:
+        import shapely._buildcfg
+        geos = '{}.{}.{}'.format(*shapely._buildcfg.geos_version)
+        geos_dir = shapely._buildcfg.geos_library_path
+    except ImportError:
+        geos = None
+        geos_dir = None
+
+    try:
+        import fiona
+        gdal = fiona.env.get_gdal_release_name()
+        gdal_dir = fiona.env.GDALDataFinder().search()
+    except ImportError:
```

Comment:
maybe need some other errors to catch here as well, as I suppose older versions of fiona don't have this

Reply:
Also for the others, it's maybe safer to just do `except Exception` in all cases (it's not that important here to catch the correct error, it's not actual library code)